Scripts/SetIfEmpty/SetIfEmpty.yml

```diff
@@ -0,0 +1,40 @@
+args:
+- default: false
+  description: The string or dictionary value to check if empty.
+  isArray: false
+  name: value
+  required: false
+  secret: false
+- default: false
+  description: The new value to return if the original value was empty.
+  isArray: false
+  name: defaultValue
+  predefined:
+  - ''
+  required: true
+  secret: false
+- auto: PREDEFINED
+  default: true
+  defaultValue: 'true'
+  description: If set to true, empty strings and dictionaries will be identified as
+    invalid values and the default value will be returned. True by default.
```

Comment:
- `be identified as invalid values` - the values are not invalid, maybe something like `be handled as empty value
- no need to write `default is true`